src/pydap/parsers/dmr.py

```diff
@@ -169,35 +177,80 @@ def dmr_to_dataset(dmr):
         for dim in variable["dims"]:
             variable["shape"] += (variables[dim]["size"],)
 
-    # Convert the ordered dictionary to dataset
-    dataset_name = dom_et.attrib["name"]
-    dataset = pydap.model.DatasetType(dataset_name)
     for name, variable in variables.items():
-        data = DummyData(dtype=variable["dtype"], shape=variable["shape"])
+        var_name = variable["name"]

```

Comment:
This makes it possible to separate the variable name from a path-like variable name like `Group1/Group2/var_name`. path is later passed as an attribute to the specific variable, which is then used to correctly assign Group behavior in the dataset